src/legacy/ui/field_formats/content_types/html_content_type.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { escape, isFunction } from 'lodash';
+import { FieldFormatConvert, IFieldFormat } from '../types';
+
+// @ts-ignore
+import { asPrettyString } from '../../../core_plugins/kibana/common/utils/as_pretty_string';
+// @ts-ignore
+import { getHighlightHtml } from '../../../core_plugins/kibana/common/highlight/highlight_html';
+
+type HtmlConventTypeConvert = (
```

Comment:
Could we TS this better?

Reply:
I've added interface for Field property.